Snake/BusinessLayer/GameSound.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Media;
+using System.IO;
+
+namespace BusinessLayer
+{
+    class GameSound
+    {
+        private string path = System.IO.Directory.GetCurrentDirectory();
```

Comment:
don't fully qualify a class name with the namespace unless you need to resolve a conflict with another class